admin_guide/seccomp.adoc

```diff
@@ -0,0 +1,127 @@
+allowing administrators greater control over the security of workloads running
+in {product-title}.
+
+Seccomp support is achieved via two annotations in the pod configuration:
+
+* *seccomp.security.alpha.kubernetes.io/pod*: profile applies to all containers in the pod that do not override
+* *container.seccomp.security.alpha.kubernetes.io/<container name>*: container-specific profile override
+
+[IMPORTANT]
+====
+Containers are run with *unconfined* seccomp settings by default.
+====
+
+For detailed design information, refer to the
+link:https://github.com/kubernetes/kubernetes/blob/release-1.4/docs/design/seccomp.md[seccomp
+design document].
+
+[NOTE]
+====
+Seccomp is a feature of the Linux kernel. To ensure seccomp is enabled on your
+system run:
+
+----
+$ cat /boot/config-`uname -r` | grep CONFIG_SECCOMP=
+CONFIG_SECCOMP=y
+----
+====
+
+[[seccomp-configuring-openshift-for-seccomp]]
+== Configuring {product-title} for Seccomp
+
+. Create the seccomp profile. 
++
+A seccomp profile is a json file providing syscalls and the appropriate action
+to take when a syscall is invoked. The
+link:https://github.com/docker/docker/blob/master/profiles/seccomp/default.json[default
+profile] is sufficient in many cases, the cluster administrator must define the
+security constraints of an individual system.
++
+To create your own custom profile, create a file on every node in the
+`seccomp-profile-root` directory.
++
+If you are using the default *runtime/default* profile, you do not need to
+create one.
+
+. Configure your nodes to use the *seccomp-profile-root* where your profiles
+will be stored. In the *node-config.yaml* via the *kubeletArguments*:
++
+----
+kubeletArguments:
+  seccomp-profile-root:
+    - "/your/path"
+----
+
+. Restart the node host.
+
+. In order to control which profiles may be used and to set the default profile,
+xref:../architecture/additional_concepts/authorization.adoc#authorization-seccomp[configure
+your SCC] via the *seccompProfiles* field. The first profile will be used as a
+default.
++
+When configuring seccomp profiles to use, follow these formatting rules:
+
+* *runtime/default*: the default profile for the container runtime (no profile on disk required)
+* *unconfined*: unconfined profile, ie, no seccomp sandboxing
+* *localhost/<profile-name>*: the profile installed to the node's local seccomp profile root
++
+For example, if you are using the default *runtime/default* profile, configure the *restricted* SCC with:
++
+----
+seccompProfiles:
+- runtime/default
+----
+
+[[seccomp-configuring-openshift-with-custom-seccomp]]
+==  Configuring {product-title} for a Custom Seccomp Profile
+
+To ensure pods in your cluster run with a custom profile in the *restricted* SCC:
+
+. Create the seccomp profile in *seccomp-profile-root*:
++
+----
+kubeletArguments:
+  seccomp-profile-root:
+    - "/your/path"
+----
+
+. Configure *seccomp-profile-root*:
++
+----
+seccompProfiles:
+- runtime/default
```

Comment:
this would be the `localhost/<name>` option if you wanted to use a custom profile.  If you're just using the `runtime/default` you would not need to set the seccomp-profile-root.